apps/backend/src/datasources/postgres/AdminDataSource.ts

```diff
@@ -215,8 +215,8 @@ export default class PostgresAdminDataSource implements AdminDataSource {
     return database
       .select()
       .from('users')
-      .where('organisation', id)
-      .then((users: UserRecord[]) =>
+      .where('institution_id', id)
+      .then((users: Array<UserRecord & InstitutionRecord>) =>
```

Comment:
Nice catch. I ended up removing it as you suggested. That type as it wasn't used anywhere. 
I also think that type residing in postgres records file wasn't very usable, because it had separate properties for user with type User and institution with type Institution, whereas the results from Postgres are returned back as User & Institution as a union when using postgres .join() function.